tools/bin/commands/build

```diff
@@ -21,6 +21,7 @@ build::usage() {
 -e  --ensure                  If building a Go module, run 'dep ensure' before building
 -l  --local                   For Go builds: Use a locally installed go, otherwise run the Go build from a container
 -c  --clean                   Run clean builds (mvn clean)
+    --resume <m1>             Resume build from maven module (mvn -rf m1)
```

Comment:
@dhirajsb could you please add this to the documentation, too ?

Reply:
@rhuss will do

Reply:
Looks like it's been updated already.